src/timeline.ts

```diff
@@ -42,10 +40,19 @@ class Timeline implements IAppView {
   private build() {
     // TODO build timeline using D3 of parts that doesn't change on update()
     this.$node.html(`
-      <!--<h3>${Language.TIMELINE}</h3>-->
-      <ul class="output"></ul>
-      <div id="ratioBar" class ="ratioBarChart"></div>
       <div id="timeline"></div>
+      <div id="nav-bar">
+            <button type="button" class="btn-nochanges" data-toggle="buttons-checkbox">No Changes</button>    
+            <button type="button" class="btn-removed" data-toggle="buttons-checkbox">Removed</button>
+            <button type="button" class="btn-added" data-toggle="buttons-checkbox">Added</button>
+            <button type="button" class="btn-content" data-toggle="buttons-checkbox">Content</button> 
+                    
+            <button type="button" class="btn btn-primary" data-toggle="buttons-checkbox">Show/Hide Timeline</button>
```

Comment:
Add additional CSS class `.toggleTimeline` and use it as selector for D3 instead of `.btn-primary`